alcs-api/src/common/automapper/reconsideration.automapper.profile.ts

```diff
@@ -88,6 +88,10 @@ export class ReconsiderationProfile extends AutomapperProfile {
           (a) => a.card,
           mapFrom((rd) => this.mapper.map(rd.card, Card, CardDto)),
         ),
+        forMember(
```

Comment:
Do you really need this one? Docs mentions that automapper is able to understand boolean | null

Reply:
Yes, without this it is no longer mapping and doesnt return the property at all.